src/containers/User/UserArea.tsx

```diff
@@ -33,6 +34,16 @@ const UserArea: React.FC = () => {
   } = useListTemplates(templatePermissions, false)
   const { dataViewsList } = useDataViewsList()
   const { intReferenceDocs, extReferenceDocs } = useReferenceDocs(currentUser)
+  const [active, setActive] = useState(false)
```

Comment:
Could we name this `hamburgerActive` so it's clear what it's for?

Reply:
Done